spec/lib/miq_automation_engine/miq_ae_event_spec.rb

```diff
@@ -183,5 +183,32 @@ module MiqAeEventSpec
         end
       end
     end
+
+    describe '.build_evm_event' do
+      let(:host) { FactoryGirl.create(:host, :ext_management_system => ems) }
```

Comment:
I would recommend using a sub-class of the Host model to validate we pass the base class names to `deliver_queue`.